src/invoker.js

```diff
@@ -22,15 +22,13 @@ var curryN = require('./curryN');
  *
  *      var sliceFrom = R.invoker(1, 'slice');
  *      sliceFrom(6, 'abcdefghijklm'); //=> 'ghijklm'
- *      var sliceFrom6 = R.invoker(2, 'slice', 6);
+ *      var sliceFrom6 = R.invoker(2, 'slice')(6);
  *      sliceFrom6(8, 'abcdefghijklm'); //=> 'gh'
  */
-module.exports = curry(function invoker(arity, method) {
-  var initialArgs = _slice(arguments, 2);
-  var len = arity - initialArgs.length;
-  return curryN(len + 1, function() {
-    var target = arguments[len];
-    var args = initialArgs.concat(_slice(arguments, 0, len));
+module.exports = _curry2(function invoker(arity, method) {
+  return curryN(arity + 1, function() {
+    var target = arguments[arity];
+    var args = _slice(arguments, 0, arity);
     return target[method].apply(target, args);
```

Comment:
I almost did it ;) But you're right. Readability is probably not harmed by doing so.